shared/domain/src/main/java/gov/va/vro/model/AbdEvidenceWithSummary.java

```diff
@@ -21,4 +21,6 @@ public class AbdEvidenceWithSummary {
   private Map<String, Object> evidenceSummary;
 
   private String errorMessage;
+
+  private boolean sufficientForFastTracking;
```

Comment:
@jbregstone @rajesh-Krishnasamy I added this to capture the result of the sufficiency calculation